templates/include/telemetry/histogram.1.schema.json

```diff
@@ -33,7 +33,13 @@
       "minimum": 0
     },
     "values": {
-      "type": "object"
+      "type": "object",
+      "patternProperties": {
+        "^[0-9].*$": {
```

Comment:
I believe this would allow keys like "3foo". For all-digits (and disallowing empty keys), you'd need something like `"^[0-9]+$"`.